src/hooks/AllPaymentHooks.res

```diff
@@ -371,6 +373,7 @@ let useRedirectHook = () => {
   let (allApiData, setAllApiData) = React.useContext(AllApiDataContext.allApiDataContext)
   let redirectioBrowserHook = useBrowserHook()
   let retrievePayment = useRetrieveHook()
+  let handleSuccessFailure = useHandleSuccessFailure()
```

Comment:
please don't consume handleSuccessFailure hook here, we are already consuming useHandleSuccessFailure hook in files like Redirect.res and making responseCallback and errorCallback there and passing it as a prop to useRedirect hook. 

Please use responseCallback and errorCallback based on your use case instead of directly using handleSuccessFailure